spec/models/comment_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  user = User.new(name: 'najib', posts_counter: 0)
+  post = Post.new(title: 'man', comments_counter: 0, likes_counter: 0)
+  it 'is vlaid with a user and a post' do
+    cm = Comment.new(user:, post:)
+    expect(cm).to be_valid
+  end
+
+  it 'is invalid without user' do
+    cm = Comment.new(post:)
+    expect(cm).to be_invalid
+  end
+
+  it 'the comment sould not be present' do
+    Comment.new(user:, post:)
+    fountp = Post.find_by(id: post.id)
+    expect(fountp).to_not be_present
+  end
+
+  it 'the comment sould not be present' do
+    Comment.new(user:, post:)
+    fountp = Post.find_by(id: post.id)
+    expect(fountp).to_not be_present
+  end
+  # method test
+  it 'comment counter method' do
+    user = User.create(name: 'najib', posts_counter: 0)
+    post = Post.create(title: 'man', comments_counter: 0, likes_counter: 0)
+    Comment.create(user:, post:)
+    expect(post.comments_counter).to eq('0')
+  end
+end
```

Comment:
- [ ] Just as the previous reviewer mentioned, kindly ensure that there is a test for the `update_comments_counter` method to ensure that it works as expected. The test needs to check that creating a new comment calls the `update_comments_counter` method and increments the `comments_counter` by 1. 👍

- Your Method
![image](https://github.com/najibullahjafari/ruby_social_media/assets/102699958/ed15c32a-ccd5-412c-9fe3-10e9e9df3743)